### PR TITLE
fix: avoid dist upgrade for cvm to avoid nullboot issue

### PR DESCRIPTION
**Pull request by anujmaheshwari1**
**What type of PR is this?**
This PR skips doing a dist-upgrade for CVM to avoid nullboot TPM issues if kernel isnt latest.

Green build for all 20.04 images https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=72765092&view=results

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh [hunk 1]
```diff
@@ -21,7 +21,6 @@ installDeps() {
 
     aptmarkWALinuxAgent hold
     apt_get_update || exit $ERR_APT_UPDATE_TIMEOUT
-    apt_get_dist_upgrade || exit $ERR_APT_DIST_UPGRADE_TIMEOUT
```
Comment (cameronmeissner) [inline parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh:24]: can we get away with only doing this for cvm?

Comment (cameronmeissner) [reply parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh:24]: oh nvm cant read

**Review by cameronmeissner — APPROVED** [review]

File: vhdbuilder/packer/install-dependencies.sh [hunk 2]
```diff
@@ -21,7 +21,7 @@ echo "Components downloaded in this VHD build (some of the below components migh
 
 # fix grub issue with cvm by reinstalling before other deps
 # other VHDs use grub-pc, not grub-efi
-if [[ "${UBUNTU_RELEASE}" == "20.04" ]]; then
+if [[ "${UBUNTU_RELEASE}" == "20.04" ]] && [[ "$IMG_SKU" != "20_04-lts-cvm" ]]; then
```
Comment (alexeldeib) [inline vhdbuilder/packer/install-dependencies.sh:24]:
```suggestion
if [[ "${UBUNTU_RELEASE}" == "20.04" ]] && [[ "$IMG_SKU"=!= "20_04-lts-cvm" ]]; then
```

you have this backwards if you're trying to handle fips

Comment (alexeldeib) [reply vhdbuilder/packer/install-dependencies.sh:24]: whoops typo. fixed.

**Pull request closed without merging**